verticapy/tests/vModel/test_elastic_net.py

```diff
@@ -148,10 +151,15 @@ def test_to_sklearn(self, model):
         prediction = model.cursor.fetchone()[0]
         assert prediction == pytest.approx(md.predict([[3.0, 11.0, 93.0]])[0][0])
 
-    @pytest.mark.skip(reason="shap doesn't want to work on python3.6")
-    def test_shapExplainer(self, model):
-        explainer = model.shapExplainer()
-        assert explainer.expected_value[0] == pytest.approx(5.81837771)
+    try:
+        import shap
+
+        def test_shapExplainer(self, model):
+            explainer = model.shapExplainer()
+            assert explainer.expected_value[0] == pytest.approx(5.81837771)
+
+    except:
+        pass
```

Comment:
The same here. Please use `pytest.skip(reason)` instead of `pass`. The reason can be something like "unsupported configuration".